cartesi-rollups/build-dapps/dapp-host-mode.md

```diff
@@ -12,7 +12,19 @@ When running in host mode, localhost port `5004` will be used by default to allo
 
 ## Step 1: Run the environment
 
-The first step is to run the environment in host mode using the following command:
+The first step is to run the environment in host mode. Before this, check if your Docker supports `riscv` by running:
```

Comment:
Like Arthur said, this section doesn't make too much sense under 'host mode'.

Reply:
@nicocartesi can we make the necessary changes for this please based on the feedback from Arthur and Gabriel?